### ISSUE TITLE
how to configure Redis as cache?

### ISSUE THREAD

**Issue by glenk** [zmartzone/ngx_openidc_module#9]
I'd like to use Redis for the cache backend (instead of shared memory).   Is the OpenIDCCache setting implemented?  If so, what is the format (and valid parameters) for this setting?

**Comment by zandbelt** [comment]
yes, `OpenIDCCache` should work; sorry for the lack of documentation, feel free to add if you want to...
